randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1417,10 +1417,10 @@ Extra - asset_id: collision_camera_018
   * Extra - right_shield_def: None
   > Dock to Big Hub
       Any of the following:
-          Flash Shift or Flash Shift Skip (Beginner) or Lay Cross Bomb or Use Spin Boost
+          Flash Shift or Flash Shift Skip (Beginner)
```

Comment:
Either this one needs to be changed to include Cross and Spin with Flash Skip, or the Artaria one needs changed to just be Flash Skip. And if you choose changing this one, should probably include Movement (Beginner)